### PR TITLE
{chem}[dummy] ORCA 4.0.1 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**
This version is a bugfix of `4.0.0`

### CODE REVIEW

File: easybuild/easyconfigs/o/ORCA/ORCA-4.0.1-OpenMPI-2.0.2.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single-
+ and multireference correlated ab initio methods.
+ It can also treat environmental and relativistic effects."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# Download from https://cec.mpg.de/orcadownload/index.php
+sources = ['%%(namelower)s_%s_linux_x86-64_openmpi%s.tar.xz' % (version.replace('.', '_'), ompi_ver.replace('.', ''))]
+
+checksums = ['49f4e155a5425d9be2b1e90bf2fa7b61']
```
Comment (boegel) [inline easybuild/easyconfigs/o/ORCA/ORCA-4.0.1-OpenMPI-2.0.2.eb]: @hajgato We're slowly moving towards SHA256 checksum, so let's replace this with:

```python
checksums = ['6644b95fdb16e117b870fd9e5cea8b4f829f1547d5df4e87d52ef08e3f1ee310']
```

(do verify on your end)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2115.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/db926923164871888ab5be9adbb14d96 for a full test report.

File: easybuild/easyconfigs/o/ORCA/ORCA-4.0.1-OpenMPI-2.0.2.eb (hunk 1 above)
Comment (hajgato) [inline easybuild/easyconfigs/o/ORCA/ORCA-4.0.1-OpenMPI-2.0.2.eb]: @boegel Done!

**Comment by hajgato** [comment]
@boegel sha256 checked on my side.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2451.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f54b2855ea4a094fb484c69995ca3809 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**